sphinx/source/docs/user_guide/server.rst

```diff
@@ -54,112 +57,104 @@ this.
+You might also want to use the Bokeh server to publish interactive data
+visualizations and applications to a wider audience, say, on the internet
+or an internal company network. The Bokeh server also suits this usage well,
+but you might want to first consult the following:
 
-* :ref:`userguide_server_applications` - how to create Bokeh Applications, and then refer to the section
-* :ref:`userguide_server_deployment` - how to deploy the Bokeh server with your application.
+* For information on how to create Bokeh applications, see
+  :ref:`userguide_server_applications`.
+* For information on how to deploy a server with your application, see
+  :ref:`userguide_server_deployment`.
 
 
 .. _userguide_server_use_case_shared:
 
-Shared Publishing
+Shared publishing
 ~~~~~~~~~~~~~~~~~
 
-Both of the scenarios above involve a *single creator* making applications
-on the server, either for their own local use or for consumption by a
-larger audience. Another scenario is the case where a group of *several
-creators* all want to publish different applications to the same server. **This
-is not a good use-case for a single Bokeh server.** Because it is possible to
-create applications that execute arbitrary Python code, process isolation and
-security concerns make this kind of shared tenancy prohibitive.
-
-In order to support this kind of multi-creator, multi-application environment,
-one approach is to build up infrastructure that can run as many Bokeh servers
-as-needed, either on a per-app, or at least a per-user basis. It is possible
-that we may create a public service to enable just this kind of usage in the
-future, and it would also certainly be possible for third parties to build
-their own private infrastructure to do so as well, but that is beyond the
-scope of this User's Guide.
-
-Another possibility is to have a single centrally created app (perhaps by an
-organization), that can access data or other artifacts published by many
-different people (possibly with access controls). This sort of scenario *is*
-possible with the Bokeh server, but often involves integrating a Bokeh
-server with other web application frameworks.
+Both of the scenarios above involve *one person* making applications on the
+server, either for personal use or for consumption by a larger audience.
+
+While it is possible for *several people* to publish different applications
+to the same server, **this does not make for a good use case** because hosted
+applications can execute arbitrary Python code. This raises process isolation
+and security concerns and makes this kind of shared tenancy prohibitive.
+
+One way to support this kind of multi-application environment with multiple
+users is to build up infrastructure that can run a Bokeh server for each app or
+at least for each user. The Bokeh project or a third party might create a
+public service for this kind of usage in the future but such developments are
+beyond the scope this article.
```

Comment:
Reads weird to me, I don't really think of parts of a projects documentation as an "article". Maybe "the project documentation" or just the original "this User's Guide"

Reply:
An easy solution would be to just say "this document". 
You could also use "this documentation", I think. In my mind, the user guide has sections, so another option would be "section of the user guide".